platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/PolicyManager.java

```diff
@@ -104,7 +104,27 @@ public void setContextPolicy(String path, ContextPolicy contextPolicy) {
         policyStore.put(path, contextPolicy);
     }
 
+    /**
+     * Initializes the policy store. This method will be called every time the policy attributes
+     * change. This will happen after the component has been initialized (see {@link #configure()}
+     * and when an update is made to the {@code org.codice.ddf.security.policy.context.impl.PolicyManager}
+     * configuration pid.
+     * <br/>
+     * See https://osgi.org/javadoc/r6/cmpn/org/osgi/service/cm/ManagedService.html for more
+     * details on how and when this method may be called.
+     *
+     * @param properties map of properties to use to initialize the policy store.
+     *                   Since there is no configuration file bound to these properties by default,
+     *                   this map may be {@code null}.
+     */
     public void setPolicies(Map<String, Object> properties) {
+        if (properties == null) {
+            LOGGER.info("setPolicies() called with null properties map. "
+                    + "Policy store should have already been initialized so ignoring.");
```

Comment:
Based on the content of this log message, I think this should be at a debug level.